src/credential/credentialController.ts

```diff
@@ -98,12 +98,12 @@ export class CredentialController {
         this.eventStream.post(new CredentialReset());
     }
 
-    private async refreshCredential(): Promise<void> {
+    private async refreshCredential(correlationId: string): Promise<void> {
         let userInfo = await this.keyChain.getUserInfo();
         if (userInfo) {
             this.signInStatus = 'SignedIn';
             this.userInfo = userInfo;
-            this.eventStream.post(new CredentialRetrievedFromLocalCredentialManager(this.credential));
+            this.eventStream.post(new CredentialRetrievedFromLocalCredentialManager(correlationId, this.credential));

```

Comment:
possible to reuse `UserSignInCompleted` even with `RetrievedFromCache`, like telemetry sending `SignIn.Completed`